webapp/web/.gcloudignore

```diff
@@ -1,28 +1,35 @@
-# Don't upload test data, package config, etc. to AppEngine.
-**/*_test.go
+# Tests
```

Comment:
```suggestion
# Files matching the following rules won't be uploaded to AppEngine (webapp).
# The syntax is the same as `.gitignore`; we do not use `.gitignore` directly
# because we need to update some generated files.

# Tests
```